blog-content/articles/testing-makes-you-faster-day-one.md

```diff
@@ -155,5 +160,83 @@ class ChestTest {
 }
 ``` 
 
-This is the untapped power of tests I was talking about at the beginning - the power to exercise any piece of your software in `O(1)` keystrokes.
-Regardless of how large or small you software is, tweaking your castle and validating it in a matters of seconds using an ever growing system of teleports will make you consistently productive on day one, day two, day hundred and day thousand. 
+This is the untapped power of tests I was talking about at the beginning - the power to exercise any piece of your software in one button press.
+Regardless of how large or small you software is, tweaking your castle and validating it in a matters of seconds using 
+an ever growing system of teleports will make you consistently productive on day one, day two, day hundred and day thousand. 
+
+# Real Life Boring Example
```

Comment:
This boring example is night and day. It precisely describes what a teleport is and why it helps. With this change (even without the change in the summary I recommended above) this article is ready to be published! Very impressive.